index.html

```diff
@@ -15,11 +16,33 @@
 </head>
 
 <body>
+  <div class="container home">
 
-  
      
-  <div class="container home">
-   
+      <div class ="navContainer">
+          <nav> <!--Navigation Bar-->
+            <div class="navItem"><h2>Fun Bus</h2></div> <!--Fun Bus-->
+            <div class="navItem navAnchors">
+              <!--Anchors-->
+              <a href="index.html">Home</a>
+              <a href="#">About Us</a>
+              <a href="#">Blog</a>
+              <a href="#">Contact</a>
+              <!--Anchors-->
+            </div>
+          </nav>
+        </div>
+
+    <div class="test">
+      <div> 
+        <img src="/img/fun-bus.jpg" alt="This is a picture of a bus"> 
```

Comment:
Beware of leading and trailing slashes that might affect the source of your image tags.